### PR TITLE
Fixing #895. Just adding a null check before accessing ProjectHelper.ParsedProject

### PR DESCRIPTION
**Pull request by amanda-tarafa**
Quick PR so as to avoid this bug interfering with everyone else's work. Will add tests at a later PR.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/GenerateConfigurationCommand/GenerateConfigurationContextMenuCommand.cs [hunk 1]
```diff
@@ -160,7 +160,7 @@ private void OnBeforeQueryStatus(object sender, EventArgs e)
             }
 
             // Ensure that the menu entry is only available for ASP.NET Core projects.
-            var selectedProject = SolutionHelper.CurrentSolution.SelectedProject.ParsedProject;
+            var selectedProject = SolutionHelper.CurrentSolution.SelectedProject?.ParsedProject;
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/GenerateConfigurationCommand/GenerateConfigurationContextMenuCommand.cs]: Can ```CurrentSolution``` ever be null?

Comment (amanda-tarafa) [reply GoogleCloudExtension/GoogleCloudExtension/GenerateConfigurationCommand/GenerateConfigurationContextMenuCommand.cs]: No, I checked by running VS, even if there's no solution open DTE returns some sort of empty solution.
Also, before my previous PR the code was exactly as is now execpt for the call to ParsedProject:
var selectedProject = SolutionHelper.CurrentSolution.SelectedProject;
and current solution being null would have been a problem there if it was possible.
I can add the null check just to be on the super safe side.

Comment (iantalarico) [reply GoogleCloudExtension/GoogleCloudExtension/GenerateConfigurationCommand/GenerateConfigurationContextMenuCommand.cs]: No this should be fine.  We can double check this all with the tests when we get them written.

**Comment by iantalarico** [comment]
Thanks for the quick fix!

**Pull request merged**